freud/order.pyx

```diff
@@ -272,6 +286,22 @@ cdef class Hexatic(_PairCompute):
     def compute(self, system, neighbors=None):
         R"""Calculates the hexatic order parameter.
 
+        Example::
+
+            >>> # Assuming user has defined k and box, points/positions,
+            >>> # and system(box, points) have already been defined
+            >>> # Compute the K-atic order for the 2 D system
+            >>> # Recommend visualizing data with the particle_property below
+            >>> import freud
+            >>> from ovito.data import *
+            >>> import numpy as np
+            >>> order_param = freud.order.Hexatic(k)
```

Comment:
We may want to pick a more descriptive name here as well. Something like `hex_order` if we choose `k=6` or `k_order` for more generality.